### ISSUE TITLE
A problem with all.gbk.tar.gz download from NCBI ftp

### ISSUE THREAD

**Issue by minsookim1983** [dparks1134/FragmentClassificationPackage#2]
Hello,

This is Min-Soo Kim who is interested in your FCP. I am trying to do local installing the FCP, but I got an error at the step for NCBI bacterial and archaeal genome download.

I realized that the whole structure of NCBI ftp site was recently changed, and there is not a file named "all.gbk.tar.gz" any more in the new version of NCBI ftp site. This is why I've got the error when NCBI bacterial and archaeal genome downloading.

Please, let me know how to correct the script or directory path written in the FCP_install.py file for NCBI bacterial and archaeal genome database. One thing more I want is how to update new RefSeq genomes on the FCP.

Min-Soo Kim

**Comment by donovan-h-parks** [comment]
Hello,

Unfortunately, these is not an easy way to update the FCP_install.py script as NCBI no longer provides a single file with all bacterial and archaeal genomes. FCP is also no longer being actively maintained since it is rather old at this point and many alternatives are now available (e.g., Kraken, MyTaxa). 

However, if you wish to use the FCP you can find information on downloading all genomes at https://www.ncbi.nlm.nih.gov/genome/doc/ftpfaq/. The install script would then need to be modified to reflect the location of these genomes.

Sorry I can't be of more direct assistance.

Regards,
Donovan

**Comment by glucksfall** [comment]
Obviously, I'm tooooo late to reply to this issue. I have also been looking for that file and I found it on ftp://ftp.ncbi.nlm.nih.gov/genomes/archive/old_refseq/Bacteria/

regards,
